source/js/components/petition/petition.jsx

```diff
@@ -152,6 +151,9 @@ export default class Petition extends React.Component {
       let postalCode = this.postalCode && this.postalCode.element.value;
       let comment = this.comment && this.comment.element.value;
       let newsletterSignup = false;
+      let locale = document
```

Comment:
This code was introduced several times, so instead, let's change `petition/locales.js` so that it exposes something like:
```
export {
  getCurrentLanguage, // this is currently getCurrentLocale, but we should rename this function
  getText: get
};
```
and then make getCurrentLocale tap into the meta tag if it exist with its current URL code as fallback code instead.

Reply:
Let's also make sure to call this `lang`, since we're accessing the "wagtail:language" meta element